src/colornames.csv

```diff
@@ -27340,6 +27426,7 @@ Trellised Ivy,#9aa097,
 Trench,#af9770,
 Trendy Green,#7e8424,
 Trendy Pink,#805d80,
+Trenton Taupe,#a39788,

```

Comment:
Looks like this is a place. Not a very famous one :D I would be ok with remove that one. Don't have strong feelings about it though. Up to you